logeco_custom/models/res_partner.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Coop IT Easy SCRL fs
+#   Manuel Claeys Bouuaert <manuel@coopiteasy.be>
+# Inspiration from OCA/project/project_task_code
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models, _
+import logging
+
+
+_logger = logging.getLogger(__name__)
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    ref = fields.Char(
+        required=True,
+        default='/',
```

Comment:
This is a strange default value but I guess it has some meaning.

Reply:
Default value comes from inspiration of `OCA/project/project_task_code` and is of no particular importance. I'll explain you live!